packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -164,32 +173,55 @@ export class SOQLEditorInstance {
         this.updateTextDocument(this.document, soql);
         break;
       }
+      case MessageType.UI_TELEMETRY: {
+        const { unsupported, errors } = e.payload as TelemetryModelJson;
+        if (errors) {
+          trackError('syntax_error', JSON.stringify(e.payload)).catch(
+            console.error
+          );
+        }
+        if (unsupported) {
+          // no need to duplicate.  unsupported and errors often both present
+          if (!errors) {
+            trackError('syntax_unsupported', JSON.stringify(e.payload)).catch(
+              console.error
+            );
+          }
+          channelService.appendLine(`This syntax is not yet supported.`);
```

Comment:
@ekapner This appears in the channel output.  

<img width="440" alt="Screen Shot 2020-12-07 at 10 00 52 PM" src="https://user-images.githubusercontent.com/599418/101443486-bffd3300-38da-11eb-8b08-80412aedcac5.png">

Reply:
`This syntax is not yet supported in SOQL Builder.  Use a text editor.` - From Emily